site/src/docs-components/typography/TypographyShowcase.jsx

```diff
@@ -34,55 +34,55 @@ const TypographyShowcase = () => (
       name="Page Title (H1)"
```

Comment:
Check out the page this component is used: https://cultureamp.design/guidelines/typography/

I think it would be clearer if we left this page alone (or even removed it)

Reply:
Yeah, I think removing it is a good idea 👍